parsec/core/fs/storage/workspace_storage.py

```diff
@@ -419,6 +426,12 @@ async def mark_prevent_sync_pattern_fully_applied(self, pattern: Pattern[str]) -
         await self.manifest_storage.mark_prevent_sync_pattern_fully_applied(pattern)
         await self._load_prevent_sync_pattern()
 
+    async def chunk_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.chunk_storage.get_local_chunk_ids(chunk_id)
+
+    async def block_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.block_storage.get_local_chunk_ids(chunk_id)
```

Comment:
Maybe those methods should be called `get_local_chunk_ids` and `get_local_block_ids`?